README.md

```diff
@@ -1,6 +1,20 @@
 # crawler-configurations-examples
 
-Checkout these sample crawler configuration files
+This repository contains:
+
+- sample configuration files for [Algolia Crawler](https://www.algolia.com/products/crawler/);
+- and a configuration tester.
+
+## How to test a configuration
```

Comment:
I would put this in `tests/README.md` and have a warning saying that it's only for Algolia's employee.

Reply:
Done!